modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc

```diff
@@ -806,62 +825,81 @@ bool CCLanePostProcessor::Process(const cv::Mat &lane_map,
 
       // accept the new lane object
       (*lane_objects)->push_back(cur_object);
+      // AINFO << "Lane ID: " << count_lane_objects 
+      //       << ", C0: " << cur_object.lateral_distance;
       origin_lateral_dist_object_id.push_back(
           std::make_pair(cur_object.lateral_distance, count_lane_objects++));
       ADEBUG << "generate a new lane object from instance.";
     }
 
     // determine spatial label of lane object
+    // Sort lanes with C0 
     std::sort(origin_lateral_dist_object_id.begin(),
               origin_lateral_dist_object_id.end(),
               CompOriginLateralDistObjectID);
-    int i_l0 = -1;
+    int index_closest_left = -1;
+    int index_closest_right = -1;
     for (int k = 0; k < count_lane_objects; ++k) {
       if (origin_lateral_dist_object_id[k].first >= 0) {
-        i_l0 = k;
+        index_closest_left = k;
+        if (k > 0) {
```

Comment:
CI didn't pass 
and I think line 845 may not be correct.  maybe it should be:    if (k < count_lane_objects-1)   ?

Reply:
THank you, This is incorrect statement.